Doc/Makefile

```diff
@@ -127,7 +127,7 @@ clean:
 
 venv:
 	$(PYTHON) -m venv $(VENVDIR)
-	$(VENVDIR)/bin/python3 -m pip install -U Sphinx==2.3.1 blurb
+	$(VENVDIR)/bin/python3 -m pip install -U Sphinx==2.3.1 blurb docutils==0.17
```

Comment:
@ned-deily 
Due to: https://github.com/sphinx-doc/sphinx/issues/9841

Reply:
Sorry about that, I'm not sure how the 3.7 Doc/Makefile change slipped through the cracks. In the future, please open a separate PR and/or ping the RM rather than including an unrelated change. I've amended this PR.